xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -752,9 +809,21 @@
         <Parameter Name="buffer" Type="System.ReadOnlySpan&lt;System.Byte&gt;" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="buffer">A region of memory. This method copies the contents of this region to the current Brotli stream.</param>
+        <summary>Writes a sequence of bytes to the current Brotli stream and advances the current position within this Brotli stream by the number of bytes written.</summary>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+Use the <xref:System.IO.Compression.BrotliStream.CanWrite%2A> property to determine whether the current instance supports writing. Use the <xref:System.IO.Compression.BrotliStream.WriteAsync%2A> method to write asynchronously to the current stream.
+
+If the write operation is successful, the position within the Brotli stream advances by the number of bytes written. If an exception occurs, the position within the Brotli stream remains unchanged.
```

Comment:
Exceptions should also be documented.

Reply:
I opened issue https://github.com/dotnet/dotnet-api-docs/issues/2799 and assigned it to myself as a reminder. I will work on it in the next days. I wanted to have the basics covered first.